README.md

```diff
@@ -38,4 +38,14 @@ import requests
 r = requests.post('http://127.0.0.1:5000/decrypt', data=encrypted_data)
 
 decrypted_data = r.text
-```
\ No newline at end of file
+```
+
+## Configuration
+
+Some of important environment variables available for configuration are listed below:
+
+| Environment Variable            | Description
```

Comment:
Other repos have a `default` column. If these have defaults can we add them, and if not, for consistency, use *none*.